sentry/src/main/java/io/sentry/jvmti/Frame.java

```diff
@@ -0,0 +1,86 @@
+package io.sentry.jvmti;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+
+/**
+ * Class representing a single call frame.
+ */
+public final class Frame {
+    /**
+     * Java method this frame is for.
+     */
+    private final Method method;
+    /**
+     * Local variable information for this frame.
+     */
+    private final LocalVariable[] locals;
+
+    /**
+     * Construct a {@link Frame}.
+     *
+     * @param method Java method this frame is for.
+     * @param locals Local variable information for this frame.
+     */
+    public Frame(Method method, LocalVariable[] locals) {
+        this.method = method;
+        this.locals = locals;
+    }
+
+    public Method getMethod() {
+        return method;
+    }
+
+    public LocalVariable[] getLocals() {
```

Comment:
What's the reasoning for returning a `LocalVariable[]` instead of a `Map<String, Object>` directly here?